app/pkg/email/smtp/smtp.go

```diff
@@ -157,3 +163,12 @@ func sendMail(localName, serverAddress string, a gosmtp.Auth, from string, to []
 	}
 	return c.Quit()
 }
+
+func generateMessageID(localName string) string {
+	timestamp := strconv.FormatInt(time.Now().UTC().UnixNano(), 10)
+	buf := make([]byte, 16)
+	rand.Read(buf)
```

Comment:
`rand.Read` might throw an error, that's very unlikely to happen, but if does, we should panic. We have a top `recover` goroutine that handles panics.

```suggestion
	_, err := rand.Read(buf)
	if err != nil {
	  panic(err)
	}
```